client-spark/spark-3/src/main/scala/org/apache/spark/shuffle/celeborn/CelebornShuffleReader.scala

```diff
@@ -30,10 +30,10 @@ import org.apache.celeborn.common.CelebornConf
 
 class CelebornShuffleReader[K, C](
     handle: CelebornShuffleHandle[K, _, C],
-    startPartition: Int,
-    endPartition: Int,
     startMapIndex: Int = 0,
     endMapIndex: Int = Int.MaxValue,
+    startPartition: Int,
+    endPartition: Int,
```

Comment:
It would break gluten celeborn shuffle manager...

Reply:
😭😭😭

Reply:
Make another PR #1869 to fix 😂